test/jdk/javax/swing/UIDefaults/6795356/TableTest.java

```diff
@@ -1,50 +0,0 @@
-/*
- * Copyright (c) 2009, 2021, Oracle and/or its affiliates. All rights reserved.
- * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
- *
- * This code is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License version 2 only, as
- * published by the Free Software Foundation.
- *
- * This code is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
- * version 2 for more details (a copy is included in the LICENSE file that
- * accompanied this code).
- *
- * You should have received a copy of the GNU General Public License version
- * 2 along with this work; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
- *
- * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
- * or visit www.oracle.com if you need additional information or have any
- * questions.
- */
-
-/*
- * @test
- * @bug 6795356
- * @summary Checks that SwingLazyValue class works correctly
- * @author Alexander Potochkin
- * @run main/othervm -Djava.security.manager=allow TableTest
- */
-
-import javax.swing.*;
-import javax.swing.table.TableCellEditor;
-import java.awt.*;
-
-public class TableTest {
-
-    public static void main(String[] args) throws Exception {
-
-        KeyboardFocusManager.getCurrentKeyboardFocusManager();
-        System.setSecurityManager(new SecurityManager());
-
-        JTable table = new JTable();
-        TableCellEditor de = table.getDefaultEditor(Double.class);
-        if (de == null) {
```

Comment:
I guess we can test this without SM since it tests SwingLazyValue?

Reply:
I believe I had removed this test because it wasn't testing anything significant. But I have re-added it please re-review and let me know if it holds value to be retained.

Reply:
Fixed in https://github.com/openjdk/jdk/pull/21498/commits/d9ee496f7349cb8beaf1e696fd430f8064baee8e